src/Symfony/Component/Form/FormFactoryBuilder.php

```diff
@@ -144,6 +156,21 @@ public function getFormFactory()
     {
         $extensions = $this->extensions;
 
+        if ($this->forceCoreExtension) {
+            $hasCoreExtension = false;
+
+            foreach ($extensions as $extension) {
+                if ($extension instanceof CoreExtension) {
```

Comment:
Should we drop this instance from `$extensions`  then to not have it registered twice?

Reply:
no. WE're precisely keeping the existing one